src/emulator/auth/state.ts

```diff
@@ -29,20 +29,30 @@ export class ProjectState {
   private oobs: Map<string, OobRecord> = new Map();
   private verificationCodes: Map<string, PhoneVerificationRecord> = new Map();
   private temporaryProofs: Map<string, TemporaryProofRecord> = new Map();
-  public oneAccountPerEmail = true;
-  private authCloudFunction: AuthCloudFunction;
-  public usageMode: UsageMode = UsageMode.DEFAULT;
+  protected _oneAccountPerEmail = true;
```

Comment:
This should be abstract getter only. Underlying storage should be moved into `AgentProjectState` (since only those instances hold the value)